### PR TITLE
Factory girl test

### PR DESCRIPTION
**Pull request by erniejamison**
Tests run. I'm getting an error:

`Failure/Error: build(:game).expect be_valid`
`NoMethodError:
       undefined method`expect' for #Game:0xbd13d1d0
     # ./spec/models/game_spec.rb:5:in `block (2 levels) in <top (required)>'`

Also, I pulled the database branch and used the whole thing so the migrations are in this pull request along with everything else unfortunately.

Dependent on 
- [x] #5

### CODE REVIEW

**Comment by lasley** [comment]
Woot we're getting closer. Double check the article on the assertion, your syntax is slightly off

**Comment by erniejamison** [comment]
Ready for review. Dependent on #5 being merged.

File: Gemfile [hunk 1]
```diff
@@ -41,6 +41,9 @@ group :development, :test do
 
   # Use Rspec for TDD
   gem 'rspec-rails', '~> 3.5'
+
+  # Use FactoryGirl for TDD
```
Comment (lasley) [inline Gemfile]: You can remove this comment

**Comment by lasley** [comment]
This looks good @erniejamison - I will need to take a double check once the DB has been merged and the diff is cleaner just to be sure, but yeah looks hooked up right.

**Comment by erniejamison** [comment]
Up for review

**Comment by wdehaes** [comment]
LGTM

**Comment by lasley** [comment]
👍

**Pull request merged**